addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2755,6 +2755,27 @@ var BooleanToggle = FieldBoolean.extend({
         'click': '_onClick'
     },
 
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Adds the icon fa-check-circle if value is true else adds icon
+     * fa-times-circle
+     *
+     * @override
+     */
+    _render() {
+        this._super.apply(this, arguments);
```

Comment:
technically you should wait for the promise, as it might be async (I know it isn't, but if it ever changes, your code will break). You can use `async/await`

Reply:
👍